case-server/src/main/java/com/powsybl/caseserver/CaseService.java

```diff
@@ -99,22 +136,31 @@ Path getCaseFile(String caseName) {
         return Optional.empty();
     }
 
-    boolean caseExists(String caseName) {
+    boolean caseExists(UUID caseName) {
         checkStorageInitialization();
-
         Path caseFile = getCaseFile(caseName);
+        if (caseFile == null) {
+            return false;
+        }
         return Files.exists(caseFile) && Files.isRegularFile(caseFile);
     }
 
-    void importCase(MultipartFile mpf) {
+    UUID importCase(MultipartFile mpf) {
```

Comment:
Something bother me here: there is no way to import a case in the public part using the REST API? 
Regarding the ongoing work on FTP cron to feed case server with public data, how will data be sent to the case server?
Maybe we should add an option to the import to decide if we import it on public or private part.